test/e2e/accounts/snap-account-eth-swap.spec.ts

```diff
@@ -39,9 +39,11 @@ describe('Snap Account - Swap', function () {
   it.skip('swaps ETH for DAI using a snap account', async function () {
     await withFixtures(
       {
+        dapp: true,
+        dappPaths: ['snap-simple-keyring-site'],
```

Comment:
ℹ️  with this key, we can indicate we want a local server for the snap simple keyring site